### PR TITLE
Codify fixes

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: static/js/request_row.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import $ from 'jquery';
 import moment from 'moment';
 import {datetimeFormat} from './utils.js';
-import tooltip from 'bootstrap';
```
Comment (jnation3406) [inline static/js/request_row.js]: I don't know why this was added in the observation-portal, but I couldn't see any difference after removing it and codify complains it's unused.

File: static/js/utils.js [hunk 2]
```diff
@@ -94,21 +93,20 @@ function decimalRaToSexigesimal(deg){
...
+  let ds = 1;
+  let dec = deg;
   if(deg < 0){
     ds = -1;
-    var dec = Math.abs(deg);
-  } else {
-    var dec = deg;
+    dec = Math.abs(deg);
   }
-  var deg = Math.floor(dec)
```
Comment (jnation3406) [inline static/js/utils.js]: We had overwritten the deg variable that was passed in before... so im not sure if this equation is correct. We should double check this.

File: static/js/request_row.js (hunk 1 above)
Comment (eheinrich) [reply static/js/request_row.js]: Probably just a leftover from before that wasn't removed

File: static/js/utils.js (hunk 2 above)
Comment (eheinrich) [reply static/js/utils.js]: This looks fine to me

File: static/js/components/pressure.vue [hunk 3]
```diff
@@ -137,6 +137,21 @@
...
+          }
+        }
+        return false;
+      }
+    },
+    created: function() {
+      this.fetchData();
+    },
+    methods: {
       toSiteNightData: function() {
```
Comment (eheinrich) [inline static/js/components/pressure.vue]: If `toSiteNightData` is now a method, doesn't that break where it is used?

Comment (jnation3406) [reply static/js/components/pressure.vue]: ah I guess I should add () where we call the method...

**Pull request merged**